lib/buildTrunkPackage.nix

```
@@ -0,0 +1,44 @@
+{ mkCargoDerivation
+, trunk
+, wasm-bindgen-cli
+, binaryen
+, nodePackages
+}:
+
+{
+  # this is not called cargoArtifacts to push users
+  # away from giving us useless artifacts built
+  # for native target instead of wasm32-unknown-unknown
+  cargoArtifactsWasm
+, indexPath ? "./index.html"
+, trunkExtraArgs ? ""
+, trunkExtraBuildArgs ? ""
```

```suggestion
, trunkExtraArgs ? ""
, trunkExtraBuildArgs ? ""
, trunkIndexPath ? "./index.html"
```

Should we call this `trunkIndexPath` for consistency with the other `trunk*` parameters?

---

Makes sense to me!